src/compass/s1_geocode_slc.py

```diff
@@ -121,7 +121,7 @@ def run(cfg: GeoRunConfig):
             ctype = h5py.h5t.py_create(np.complex64)
             ctype.commit(geo_burst_h5['/'].id, np.string_('complex64'))
 
-            grid_path = f'{root_path}/CSLC/grids'
+            grid_path = f'/CSLC/data'
```

Comment:
```suggestion
            data_path = f'/CSLC/data'
```
No more `grid`s anywhere!